conf.sh

```
@@ -0,0 +1,66 @@
+However it will require around 4GB of VRAM! 
+
+deepseed = Uses more VRAM but faster
+lowvram = Uses less VRAM but slower
+regular = Middle ground of both options above
+
+If you are not sure use lowvram.
+
+EOF
+
+if [ ! -d /home/dwemer/python-tts ]; then
+        exit "XTTSv2 not installed"
```

Shellcheck (0.10.0)

[error] Can only exit with status 0-255. Other data should be written to stdout/stderr.

(SC2242)